### PR TITLE
Cache tests enabled

### PR DESCRIPTION
**Pull request by ruchim**
fix wdl + test files

### CODE REVIEW

File: src/main/scala/centaur/test/formulas/TestFormulas.scala [hunk 1]
```diff
@@ -25,6 +27,18 @@ object TestFormulas {
 
   def runSuccessfulWorkflowAndVerifyMetadata(workflow: Workflow): Test[Unit] = runWorkflowAndVerifyMetadata(workflow, runSuccessfulWorkflow)
   def runFailingWorkflowAndVerifyMetadata(workflow: Workflow): Test[Unit] = runWorkflowAndVerifyMetadata(workflow, runFailingWorkflow)
+  def runSuccessfulWorkflowAndVerifyCacheMetadata(workflow: Workflow, cacheHit: UUID): Test[Unit] = runWorkflowAndVerifyCacheMetadata(workflow, runSuccessfulWorkflow, cacheHit)
+
+  def runWorkflowAndVerifyCacheMetadata(workflow: Workflow, f: Workflow => Test[SubmittedWorkflow], cacheHitWorkflow: UUID): Test[Unit] = {
```
Comment (geoffjentry) [inline src/main/scala/centaur/test/formulas/TestFormulas.scala:32]: what about having this be the main one which takes in an `Option[UUID]`. the previous one `runWorkflowAndVerifyMetadata` could just call this w/ None (adjust names as necessary)

**Comment by geoffjentry** [comment]
👍 once we figure out what's causing problems

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/108/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by cjllanwarne** [comment]
Could you ping me when "once we figure out what's causing problems" is true?

**Comment by ruchim** [comment]
@cjllanwarne the problem is when you run travis the tests fail but they run when I run the cache spec locally. As a first step, would you mind checking the sort of content of what the centaur cache tests are checking for?

**Comment by ruchim** [comment]
@cjllanwarne tests passing and ready for real review. I'm making a Cromwell PR branched off of 'ks_liquibase_updates' (it needs those metadata changes)-- which will point at this branch of centaur to make sure that tests will remain unbroken when 'ks_liquibase_updates' is merged.

File: src/main/resources/callCachingWorkflows/cacheBetweenWF/cacheBetweenWF.wdl [hunk 2]
```diff
@@ -1,65 +1,39 @@
...
-   	input: r2 = threeAgain.rSquared
-   }
-
    output {
-        twoAgain.area
-        fourAgain.area
+        heightProduct.trapezoidalArea
    }
-
 }
```
Comment (geoffjentry) [inline src/main/resources/callCachingWorkflows/cacheBetweenWF/cacheBetweenWF.wdl]: make sure all the files end in newlines

Comment (ruchim) [inline src/main/resources/callCachingWorkflows/cacheBetweenWF/cacheBetweenWF.wdl]: 👍

**Pull request merged**